.storybook/config.js

```diff
@@ -68,25 +77,34 @@ const guardianViewports = {
     },
 };
 
+export const viewports = [375,
+    480,
+    660,
+    740,
+    980,
+    1140,
+    1300,
+]
+
```

Comment:
My future self will bow down and thank you if you added some comments with the breakpoint names here